internal/logger/slog_helper.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"log/slog"
+	"strings"
+
+	"github.com/googlecloudplatform/gcsfuse/internal/config"
+)
+
+const (
+	LevelTrace = slog.Level(-8)
```

Comment:
Let's make -8 and 12 as a severity constant and use that.

Reply:
We don't use -8 or 12 anywhere. Only these constant variables are being used. Do you think we still need to create integer constants for this?

Reply:
Moved the comments as discussed offline.